README.md

```
@@ -148,11 +172,11 @@ Set `GITHUB_AUTH_MODE` to `app`, `pat`, or `none` to select the method.
 
 For self-hosted Git servers (e.g., GitBucket, GitLab, etc.), configure:
 
-      ```bash
+      
       uv sync
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)